pkgs/development/tools/skopeo/default.nix

```diff
@@ -51,8 +49,6 @@ buildGoModule rec {
       --prefix PATH : ${stdenv.lib.makeBinPath [ fuse-overlayfs ]}
   '';
 
-  passthru.tests.docker-tools = nixosTests.docker-tools;
```

Comment:
Removing this as`skopeo` isn't used in this test.

Reply:
Skopeo is actually used. There is an image which is pulled from the Docker registry. The `dockerTools.pullImage` function uses Skopeo to pull images and it is nice to test we can build images on top of such pulled images for instance.

Reply:
The test passes with this diff applied?

```diff
diff --git a/pkgs/build-support/docker/default.nix b/pkgs/build-support/docker/default.nix
index b2c132afd74..cf54a0e0bae 100644
--- a/pkgs/build-support/docker/default.nix
+++ b/pkgs/build-support/docker/default.nix
@@ -20,7 +20,6 @@
   runCommand,
   runtimeShell,
   shadow,
-  skopeo,
   stdenv,
   storeDir ? builtins.storeDir,
   substituteAll,
@@ -91,7 +90,6 @@ rec {
       outputHashAlgo = "sha256";
       outputHash = sha256;
 
-      nativeBuildInputs = lib.singleton skopeo;
       SSL_CERT_FILE = "${cacert.out}/etc/ssl/certs/ca-bundle.crt";
 
       sourceURL = "docker://${imageName}@${imageDigest}";
diff --git a/pkgs/top-level/all-packages.nix b/pkgs/top-level/all-packages.nix
index e0db5bc2a53..b642275e0cf 100644
--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -18539,8 +18539,6 @@ in
     rcshutdown = "/etc/rc.d/rc.shutdown";
   };
 
-  skopeo = callPackage ../development/tools/skopeo { };
-
   smem = callPackage ../os-specific/linux/smem { };
 
   smimesign = callPackage ../os-specific/darwin/smimesign { };
```

Reply:
hm, because the image is already in the cache i think.

Reply:
The test should fail is you change the `sha256` of the image `dockerTools.examples.nixFromDockerHub`.

Reply:
I don't see how it is a useful test if it never fails.

Reply:
Yep, I agree this test is not really useful.